app/src/main/java/com/tsu/wordsfactory/dictionary/ui/dictionary/DictionaryFragment.kt

```diff
@@ -1,40 +1,82 @@
 package com.tsu.wordsfactory.dictionary.ui.dictionary
 
+import android.R.attr.data
+import android.annotation.SuppressLint
+import android.content.Context
 import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
+import com.tsu.wordsfactory.app.WordsApplication
 import com.tsu.wordsfactory.databinding.FragmentDictionaryBinding
+import com.tsu.wordsfactory.repository.model.Word
+import com.tsu.wordsfactory.utils.NetworkUtils
+
 
 class DictionaryFragment : Fragment() {
 
     private var _binding: FragmentDictionaryBinding? = null
 
-    // This property is only valid between onCreateView and
-    // onDestroyView.
+    // This property is only valid between onCreateView and  onDestroyView.
     private val binding get() = _binding!!
 
+    private lateinit var viewModel: ListViewModel
+
+    private val adapter = WordsAdapter()
+    private lateinit var networkUtils: NetworkUtils
+
+    @SuppressLint("SetTextI18n")
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        val homeViewModel =
-            ViewModelProvider(this).get(DictionaryViewModel::class.java)
-
         _binding = FragmentDictionaryBinding.inflate(inflater, container, false)
 
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
Try to keep this function as small as you can. Better to separate code to some functions, such `initView`, `observeViewModel` and so on